src/openMVG/sfm/pipelines/localization/SfM_Localizer.hpp

```diff
@@ -25,13 +25,12 @@ struct Image_Localizer_Match_Data
             // found through the feature matching procedure.
   
   // pt2D and pt3D have the same number of columns.
-            
   std::vector<size_t> vec_inliers; // Index mask for both pt3D and pt2D whose elements
                                    // represent the column indices of inliers in  pt2D 
-                                   // and pt3D.   
+                                   // and pt3D.
   
   // Upper bound pixel(s) tolerance for residual errors
-  double error_max = std::numeric_limits<double>::infinity();
+  double error_max = 100;
```

Comment:
It would be better to keep the infinity limit there; and make the caller set the error to 100.